### DISCUSSION TITLE
New repository announcement

### DISCUSSION THREAD

**Discussion by paula-w** [YetiForceCompany/YetiForce#27] [General]
We are extremely grateful for our community’s involvement when it comes to reporting issues in the YetiForce system. It’s what allows us to create software that meets your expectations and needs. 
In order not to lose the chance of receiving your valuable feedback, we prepared this [repository](https://github.com/YetiForceCompany/YetiForce) where you can still report bugs, submit your comments regarding our system and ideas for its development.

To answer the most frequently submitted questions about the latest versions of YetiForce, we prepared a short Q&A:

**Q: Will the source code be available on GitHub?**
A: No, due to licensing changes introduced in YetiForce 7.0, we decided to move the code to a private repository.

**Q: Is YetiForce still open source software?**
A: Yes, YetiForce remains an open source system, however, it’s dual-licensed (commercial part, and a non-commercial part). Please refer to our license for more details: https://yetiforce.com/en/license.html 

**Q: Can I still download, install, and use YetiForce for free?**
A: Yes! Downloading, installing and using YetiForce is still free.

**Q: Where can I download YetiForce and updates?**
A: The latest installation packages can be found on:
- Our website: https://yetiforce.com/en/test-download.html
- Softaculous: https://www.softaculous.com/apps/erp/YetiForce 

We plan to release smaller updates more frequently. 
Updates can be downloaded and installed using the built-in installer directly in the YetiForce system (Software configuration → Logs → Updates).
Please take a look at the update manual to make sure the process goes smoothly: https://doc.yetiforce.com/administrator-guides/logs/updates

**Q: Where can I report errors or send pull requests?**
A:  Bugs can be reported in our new repo https://github.com/YetiForceCompany/YetiForce/issues
Since the application code is no longer available in the public repository, you can submit any code fixes in this repository or send them to [hello@yetiforce.com](mailto:hello@yetiforce.com).
Security issues and fixes should be submitted according to the policy described here: https://github.com/YetiForceCompany/YetiForce?tab=security-ov-file

**Q: Where can I find information about updates?**
A: There are a few places where you can find it:
- YetiForce has a built-in mechanism called ‘System warnings’ available in the admin panel. It will alert administrators about new releases. It requires a properly configured SMTP server.
- A list of available updates can always be found in Software configuration → Logs → Updates
- Posts and notices in the new repository: https://github.com/YetiForceCompany/YetiForce

**Comment by OkayashiM** [comment]
I remember the time when you guys was scolding vtiger a lot for incompetence in communication with community, and now you gone even further,  abandoning all opensource values. You have gradually been changing license to this point, and few people long ago realized that you are preliminary focused on coins. OS community, all the noise you have created was a driving force for cashing out this idea. I believe you will realize at some point that this is like your childish "support ukraine" on github and your website. Emotions faded, bad taste residue.
This gonna drive people away from yetiforce. It is your business and your manpower, and I do understand, but still, many guys have been left with broken dreams of being part of something what looks like descent and promising project.

**Comment by kwuite** [comment]
Real shame, the CRM looks good and I was planning on installing the CRM and adapt it for a legal firm but it seems I am unable to trust this company. What a bummer :(.

**Comment by Anonymo** [comment]
Q: Is YetiForce still open source software?
A: Yes, YetiForce remains an open source system, however....
Q: Will the source code be available on GitHub?
A: No, due to licensing changes introduced in YetiForce 7.0, we decided to move the code to a private repository.

![jackiechan](https://github.com/user-attachments/assets/0c93940f-e5fb-44c3-984b-741375df5100)

**Comment by Oxymoron290** [comment]
womp womp

**Comment by Gangadharhyati** [comment]
Can we still download 7.0.4 version and install in our webserver?? While installing we are getting Installation unsuccessful in ubantu 24.0

**Reply by paula-w** [reply]
Of course you can; take a look at the 7.0.4 version announcement #122, you can find all the links there. 
As far as your installation issues are concerned, make sure your server meets all the [requirements](https://doc.yetiforce.com/en/introduction/requirements/) and follow the [installation manual](https://doc.yetiforce.com/en/introduction/installation-manual/). 
If it still fails, and you want to report it, please open a new [issue](https://github.com/YetiForceCompany/YetiForce/issues) and provide as much information as you can, as this discussion focuses solely on the new repository.

**Reply by Gangadharhyati** [reply]
I followed every steps mentioned there and i am getting the issue in the final step as Installation unsuccessful and there is no errors also 
<img width="1361" height="729" alt="installerror" src="https://github.com/user-attachments/assets/b49d28d7-a7cb-486f-b10f-2ca99122ac8a" />
This is the screenshot
